src/rabbit_stomp_processor.erl

```diff
@@ -493,6 +495,17 @@ cancel_subscription({ok, ConsumerTag, Description}, Frame,
             end
     end.
 
+delete_binding(Channel, #subscription{exclusive = Exclusive, queue = Queue, exchange = Exchange, routing_key = RoutingKey}) ->
+    case Exclusive of
+        true ->
```

Comment:
@yurkovas sorry but that's still a bit too terse. I know what exclusive queues are. It's not true that binding add any meaningful performance overhead. They can, of course, cause messages to be routed to unexpected queues/topics. If that's the intent here, we need an integration test (or multiple tests) that demonstrate it.

So please take a step back and instead of going into the gory details of implementation details try explaining the problem in STOMP user terms.